### PR TITLE
Major Release: 2.0.0

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

# Summary
- Update Gatsby to version 4
- Fix bug with Gallery
- Updates to NPM packages
- A couple of color adjustments

# Review By (Date)
- Go live

### CODE REVIEW

**Review by yvonnetangsu — APPROVED** [review]
This looks good to me @sherakama , but would this be v1.0.4?

**Comment by yvonnetangsu** [comment]
> This looks good to me @sherakama , but would this be v1.0.4?

Actually just saw the major tag. Would this be 2.0.0 then? 🤔

**Comment by sherakama** [comment]
> > This looks good to me @sherakama , but would this be v1.0.4?
> 
> Actually just saw the major tag. Would this be 2.0.0 then? 🤔

I suppose it would. Thanks.

File: gatsby-config.js [hunk 1]
```diff
@@ -117,6 +115,23 @@ module.exports = {
...
+            page.pageContext.noindex ||
+            // Exclude pages that match the "excludes" array. (default condition)
+            tools.minimatch(
+              tools.withoutTrailingSlash(tools.resolvePagePath(page)),
+              tools.withoutTrailingSlash(excludedRoute)
+            )
+          ) {
+            return true;
+          }
+        },
```
Comment (mattanglin) [inline gatsby-config.js:134]: 👍🏼  I assume this is to get around the annoying behavior of graphql erroring on queries when properties don't exist in Storyblok yet

**Review by mattanglin — APPROVED** [review]
lgtm 👍🏼

**Pull request merged**